#
# This hack is not built by default, and we are not shipping it either
#
Index: xscreensaver-git/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver-git.orig/driver/XScreenSaver.ad.in
+++ xscreensaver-git/driver/XScreenSaver.ad.in
@@ -423,7 +423,6 @@ XScreenSaver.bourneShell:		/bin/sh
 				xspirograph -root			    \n\
 @GL_KLUDGE@ GL: 				circuit -root				    \n\
 @GL_KLUDGE@ GL: 				dangerball -root			    \n\
-- GL: 				dnalogo -root				    \n\
 @GL_KLUDGE@ GL: 				engine -root				    \n\
 @GL_KLUDGE@ GL: 				flipscreen3d -root			    \n\
 @GL_KLUDGE@ GL: 				gltext -root				    \n\
